src/resources/disputes-resource.js

```diff
@@ -1,47 +1,55 @@
+/**
+* This file was auto-generated by rebilly-sdk-generator.
+* Do not make direct changes to this file.
+*/
+
+// @ts-nocheck
 import {csvHeader} from '../request-headers';
 
 export default function DisputesResource({apiHandler}) {
-    return {
-        getAll({limit = null, offset = null, sort = null, expand = null, filter = null, q = null} = {}) {
-            const params = {
-                limit,
-                offset,
-                sort,
-                expand,
-                filter,
-                q,
-            };
-            return apiHandler.getAll(`disputes`, params);
-        },
-
-        downloadCSV({limit = null, offset = null, sort = null, expand = null, filter = null, q = null} = {}) {
-            const config = {
-                params: {
-                    limit,
-                    offset,
-                    sort,
-                    expand,
-                    filter,
-                    q
-                },
-                headers: csvHeader
-            };
-            return apiHandler.download(`disputes`, config);
-        },
-
-        get({id, expand = null}) {
-            const params = {expand};
-            return apiHandler.get(`disputes/${id}`, params);
-        },
-
-        create({id = '', data, expand = null}) {
-            const params = {expand};
-            return apiHandler.create(`disputes/${id}`, id, data, params);
+  return {
+    getAll({
+      filter = null,
+      sort = null,
+      limit = null,
+      offset = null,
+      q = null,
+      expand = null,
+    } = {}) {
+      const params = {filter, sort, limit, offset, q, expand};
+      return apiHandler.getAll(`disputes`, params);
+    },
+    create({id = '', data, expand = null}) {
+      const params = {expand};
+      return apiHandler.create(`disputes/${id}`, id, data, params);
+    },
+    get({id}) {
```

Comment:
There is an `expand` missing

Reply:
I will ask about this in `pd-mentoring.`